pkg/secretless/entrypoint/entrypoint.go

```diff
@@ -50,49 +53,77 @@ func StartSecretless(params *SecretlessOptions) {
 		log.Fatalln(err)
 	}
 
-	// Exit signal listener
+	// Optional Performance Profiling
+	handlePerformanceProfiling(params.ProfilingMode, exitListener)
 
-	// Coordinates processes interested in exit signals
-	exitListener := signal.NewExitListener()
+	configChangedChan, err := newConfigChangeChan(
+		params.ConfigFile,
+		params.ConfigManagerSpec,
+		params.FsWatchEnabled,
+	)
+	if err != nil {
+		log.Fatalln(err)
+	}
 
-	// Optional Performance Profiling
+	// Health check: Initialized
+	util.SetAppInitializedFlag()
+	util.SetAppIsLive(false)
 
-	handlePerformanceProfiling(params.ProfilingMode, exitListener)
+	var allServices internal.Service
+
+	configChangedFunc := func(cfg v2.Config) {
```

Comment:
I think a good naming rule is to reserve the `Func` suffix for naming function _types_. 

If you think about it, it doesn't really make sense that you'd change your naming scheme just because you're defining a function as a local var vs doing `func blah(...` in the package.  

Thus we can stick with the normal practice of finding a descriptive verb phrase... `reloadConfig` seems appropriate here.  

Then down below it will read:
```go
logger.Debug("Got new configuration")
reloadConfig(cfg)
```
which is more natural and descriptive than:
```go
```go
logger.Debug("Got new configuration")
configChangedFunc(cfg)
```